node/node.go

```diff
@@ -247,6 +249,10 @@ func (n *Node) baseGetAllIssuedESDTs(tokenType string, ctx context.Context) ([]s
 			continue
 		}
 
+		if strings.HasPrefix(tokenName, baseESDTKeyPrefix) {
```

Comment:
This is wrong.

The correct approach would be to filter out the native token id from config only inside `sovereignNode.go`, not touching base code.

Reply:
Ok. Changed the implementation.